docs/vm/backup-restore.md

```diff
@@ -84,15 +84,11 @@ You can choose to either delete or retain the previous volumes. By default, all
 
 ### Restore a new VM on another Harvester cluster
 
-_Available as of v1.0.0_
+_Available as of v1.4.0_
 
 Users can now restore a new VM on another cluster by leveraging the VM metadata & content backup feature.
-
-:::info prerequisites
-
-You must manually configure the virtual machine images with the same name on the new cluster first, otherwise the virtual machines will be failed to recover.
-
-:::
+Harvester will handle VMImage data automatically. If a volume is based on a VMImage and there is a backup for it, Harvester will automatically sync the VMImage to backup target.
```

Comment:
Since it's handled automatically, should we remove the section too?
```
#### Upload the same VM images to a new cluster
```

Reply:
suggest changing `VMImage` to VM image for consistency across the document.

Reply:
Thanks for the suggestion. I addressed all comments.